lib/resources/datav2/entityv2.ts

```diff
@@ -345,6 +345,7 @@ const crypto_bar_mapping = {
 export interface CryptoBar {
   Symbol: string;
   Timestamp: string;
+  Exchange: string;
```

Comment:
Please add the exchange to the `RawCryptoBar` as well then I merge this PR. (`x: string;`)